opwen_email_client/webapp/views.py

```diff
@@ -16,21 +15,23 @@
 from flask import send_file
 from flask import send_from_directory
 from flask import url_for
+
+from babel import Locale
 from flask_cors import cross_origin
 from flask_login import current_user
 from flask_security.utils import hash_password
-from passlib.pwd import genword
-
 from opwen_email_client.webapp import app
 from opwen_email_client.webapp import tasks
 from opwen_email_client.webapp.actions import SendWelcomeEmail
 from opwen_email_client.webapp.config import AppConfig
 from opwen_email_client.webapp.config import i8n
 from opwen_email_client.webapp.forms.email import NewEmailForm
+from opwen_email_client.webapp.forms.register import RegisterForm
 from opwen_email_client.webapp.forms.settings import SettingsForm
 from opwen_email_client.webapp.security import login_required
 from opwen_email_client.webapp.session import Session
 from opwen_email_client.webapp.session import track_history
+from passlib.pwd import genword
```

Comment:
Github is giving me grief with a code suggestion, but the linter is also cranky here. It should read as:
```
from datetime import datetime
from datetime import timedelta
from io import BytesIO
from os import path
from typing import Iterable
from typing import Optional

from babel import Locale
from flask import Response
from flask import abort
from flask import flash
from flask import jsonify
from flask import redirect
from flask import render_template
from flask import request
from flask import send_file
from flask import send_from_directory
from flask import url_for
from flask_cors import cross_origin
from flask_login import current_user
from flask_security.utils import hash_password
from passlib.pwd import genword

from opwen_email_client.webapp import app
from opwen_email_client.webapp import tasks
from opwen_email_client.webapp.actions import SendWelcomeEmail
from opwen_email_client.webapp.config import AppConfig
from opwen_email_client.webapp.config import i8n
from opwen_email_client.webapp.forms.email import NewEmailForm
from opwen_email_client.webapp.forms.register import RegisterForm
from opwen_email_client.webapp.forms.settings import SettingsForm
from opwen_email_client.webapp.security import login_required
from opwen_email_client.webapp.session import Session
from opwen_email_client.webapp.session import track_history
```

Reply:
Just realized that the isort on my machine is sorting in different order than the isort on the ci 😅